src/onegov/pay/models/payment_providers/stripe.py

```diff
@@ -269,7 +269,7 @@ def url(self) -> str:
     def public_identity(self) -> str:
         account = self.account
         assert account is not None
-        if account.business_name:
+        if 'business_name' in account:
```

Comment:
I would probably go for `getattr(account, 'business_name', '')` since it might exist but be empty...

Reply:
done